pkg/schedule/operator/operator_controller_test.go

```diff
@@ -493,6 +493,53 @@ func (suite *operatorControllerTestSuite) TestPollDispatchRegionForMergeRegion()
 	re.Equal(0, controller.opNotifierQueue.len())
 }
 
+func (suite *operatorControllerTestSuite) TestConcurrentMergeConflict() {
+	re := suite.Require()
+
+	for range 10 {
+		opts := mockconfig.NewTestOptions()
+		cluster := mockcluster.NewCluster(suite.ctx, opts)
```

Comment:
Can we reuse the same cluster and rebuild the region metadata within the loop? Creating separate clusters inside the loop seems a bit inefficient.